src/views/ResultsPage/ResultsPage.scss

```diff
@@ -30,20 +30,33 @@
         flex-direction: row;
         border-top: 1px solid #7d7d7d;
       }
-    }
+		}
+		@media screen and (max-width: 1025px) {
+			min-height: 0px;
+		}
   }
 
   &__pager {
     border-bottom: 1px solid #7d7d7d;
     border-top: 1px solid #7d7d7d;
-    padding: 5px 0;
-    width: 100%;
+		padding: 12px 0 12px;
+		width: 100%;
 
     @media screen and (min-width: 1025px) {
       border: 0;
       width: auto;
     }
-  }
+	}
+	&__no_pages_pager{
+		padding: 5px 0;
+		width: 100%;
+
+		@media screen and (min-width: 1025px) {
+      border: 0;
+      width: auto;
+		}
+
+	}
 
```

Comment:
modify this to be a modifier of pager element like so:
```
&__pager {
  padding: 12px 0 12px;
  width: 100%;

  @media screen and (min-width: 1025px) {
    border: 0;
    width: auto;
  }
  
  &--no-pages {
    padding: 5px 0;
	 width: 100%;
  }
}
```
You will have to modify the ternary for the class declaration on the results page